bin/contract_test/src/contract_payloads.rs

```diff
@@ -0,0 +1,43 @@
+use alloy::{primitives::Signed, sol};
+use pade_macro::{PadeDecode, PadeEncode};
+
+sol! {
+    type PoolId is bytes32;
+
+    #[derive(PadeEncode, Debug)]
+    struct Asset {
+        address addr;
+        uint128 borrow;
+        uint128 save;
+        uint128 settle;
+    }
+
+    #[derive(PadeEncode, Debug)]
+    struct Pair {
+        uint16 index0;
+        uint16 index1;
+        uint16 store_index;
+        uint256 price_1over0;
+    }
+}
```

Comment:
why do we not use the ones defined in types here?

Reply:
Ah, can do, this was a very quick-and-dirty test bin I wrote that isn't expected to remain so this was some scratch work I was messing with